materials/module-1/presentation/.gitignore

```diff
@@ -0,0 +1,4 @@
+_book
+book.pdf
```

Comment:
I think we _want_ to add a pdf version of the worksheets and slide deck to think about having this as a printable resource, and one that others can use

Reply:
I will publish the PDFs but not in this directory. I am thinking as Github Release?